.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+## :computer: Changes
+
+## :squirrel: Testing instructions
+
+## :camera: Relevant screenshots or documentation links
```

Comment:
I don't think we need a relevant screenshots section for a PR (it will probably be used for bugs/errors)
# :page_facing_up: :pushpin:

Reply:
Screenshots aren't that relevant for server side things but I find it really useful for client side development.

Reply:
Makes sense!